drivers/watchdog/wdt_rpi_pico.c

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2022, Jamie McCrae
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT raspberrypi_pico_watchdog
+
+#include <hardware/watchdog.h>
+#include <hardware/structs/psm.h>
+#include <drivers/watchdog.h>
+
+#include <logging/log.h>
```

Comment:
Should be using the new `zephyr/` prefix for includes, e.g.:

```suggestion
#include <zephyr/drivers/watchdog.h>

#include <zephyr/logging/log.h>
```

Reply:
Fixed